pkg/gen/ami/ami.go

```diff
@@ -0,0 +1,206 @@
+package ami
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/Masterminds/semver"
```

Comment:
```suggestion
	"github.com/Masterminds/semver/v3"
```

Reply:
We should use semver v3 since that's the go modules version. You'll need to use `go mod tidy` after this change.

Reply:
Did this locally to clean right away.